nat46/modules/nat46-module.c

```diff
@@ -157,6 +198,21 @@ int create_nat46_proc_entry(void) {
 
 static int __init nat46_init(void)
 {
+#ifdef PROTO_NETLINK
+	struct netlink_kernel_cfg nl_cfg = {
+		.input = nat46_nl_recv_msg
+	};
+	printk("nat46: module (version %s) loaded.\n", NAT46_VERSION);
+
+	nl_sk = netlink_kernel_create(&init_net, NETLINK_USER, &nl_cfg);
+	if (!nl_sk)
+	{
+		printk(KERN_ALERT "nat46: error creating socket\n");
```

Comment:
Should be more descriptive. Mention that it is a netlink control socket.

Reply:
Thanks for the feedback.  I'll keep your suggestion in mind as I update my patch.